integration-cli/docker_api_netmode_test.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/docker/docker/runconfig"
+	"github.com/go-check/check"
+)
+
+// GH14530. Validates combinations of --net= with other options
+func checkPS(out string) bool {
+	return strings.Contains(out, "PID   USER")
+}
+
+func (s *DockerSuite) TestNetHostname(c *check.C) {
```

Comment:
this file is named `docker_api_netmode_test` but it's adding cli tests instead of api tests, which is inconsistent.